packages/@dcl/inspector/src/lib/sdk/operations/add-asset.ts

```diff
@@ -96,6 +97,11 @@ export function addAsset(engine: IEngine) {
             values.set(componentName, { ...audioSource, src: audioSource.audioClipUrl.replace('{assetPath}', base) })
             break
           }
+          case CoreComponents.VISIBILITY_COMPONENT: {
```

Comment:
This case is not necessary, this step is to make changes to the values in the components (like replacing the {self} ids or the paths with {assetPath}) but for this component there’s no need to make modifications to the values

Reply:
@cazala thanks for the review, updated!